components/mailchimp/sources/common/http-based.js

```diff
@@ -0,0 +1,114 @@
+const base = require("./base");
+const listEventTypes = require('../list-event-types.js');
+
+module.exports = {
+  ...base,
+  props: {
+    ...base.props,
+    // eslint-disable-next-line pipedream/props-description, pipedream/props-label
+    http: {
```

Comment:
I don't think the commented line is needed for `http`.
```suggestion
    http: {
```